janus_server/src/bin/aggregator.rs

```diff
@@ -90,13 +90,18 @@ async fn main() -> Result<()> {
     let vdaf = Prio3Aes128Count::new(2).unwrap();
     let verify_param = vdaf.setup().unwrap().1.first().unwrap().clone();
 
-    let database_config = tokio_postgres::Config::from_str(config.database.url.as_str())
+    let mut database_config = tokio_postgres::Config::from_str(config.database.url.as_str())
         .with_context(|| {
             format!(
                 "failed to parse database connect string: {:?}",
                 config.database.url
             )
         })?;
+    if database_config.get_password().is_none() {
+        if let Ok(password) = std::env::var("PGPASSWORD") {
```

Comment:
Specify this as part of the Options struct above? I know that we'll specify this via env var 100% of the time (well, 100% of the time we want our Postgres password to actually be secret), but I think it's worthwhile to keep a standard method of exposing configuration options & to get the documentation that comes along with using structopts.